recipe/meta.yaml

```
@@ -39,6 +39,8 @@ requirements:
     - openexr
     - libraw
     - zlib
+    - fmt
+    - tsl_robin_map
```

```suggestion
    - tsl_robin_map
  run:
    - tsl_robin_map
```
We're also going to need `tsl_robin_map` at runtime. Either for the lib or the python bindings, but probably for the lib, given that it's used in the headers (from the current CI for linux-64):
```
$SRC_DIR/src/libtexture/imagecache_pvt.h:13:10: fatal error: tsl/robin_map.h: No such file or directory
   13 | #include <tsl/robin_map.h>
      |          ^~~~~~~~~~~~~~~~~
compilation terminated.
```

---

i think actually we need to re-add it to the py- section below.